src/coffee/services/product-projections.coffee

```diff
@@ -250,7 +250,6 @@ class ProductProjectionService extends BaseService
   #   .search()
   search: ->
     @_currentEndpoint = '/product-projections/search'
-    @fetch()
```

Comment:
So something like this?

```javascript
client.productProjections.search().fetch()
client.productProjections.search().process()

// instead of
client.productProjections.search()
client.productProjections.processSearch()
```

I don't really like `search().fetch()` to be honest...feels like you are doing 2 requests.
If `search` is just used to set the endpoint, then I would rather have something like this

```javascript
client.productProjections.forSearch().fetch()
client.productProjections.forSearch().process()

// same for suggest
client.productProjections.forSuggest().fetch()
client.productProjections.forSuggest().process()
```

And then keep `search` and `suggest` for backwards compatibility.